push-messaging-and-notifications/manifest.sample.json

```diff
@@ -7,6 +7,6 @@
       }],
   "start_url": "./index.html?homescreen=1",
   "display": "standalone",
-  "gcm_sender_id": "<YOUR PROJECT NUMBER>",
+  "gcm_sender_id": "<Your Project Number from https://console.developers.google.com>",
   "gcm_user_visible_only": true
```

Comment:
This is needed for M43 compatibility, but once M44 is stable, it can go away, right? I wish there were some way to include comments in JSON files so that you could make that clear.